tests/v2/test_waku_filter.nim

```diff
@@ -2,286 +2,316 @@
 
 import
   std/[options, tables, sets],
-  testutils/unittests, chronos, chronicles,
+  testutils/unittests,
+  chronos, 
+  chronicles,
   libp2p/switch,
-  libp2p/protobuf/minprotobuf,
-  libp2p/stream/[bufferstream, connection],
   libp2p/crypto/crypto,
-  libp2p/multistream,
+  libp2p/multistream
+import
   ../../waku/v2/node/peer_manager/peer_manager,
-  ../../waku/v2/protocol/waku_filter/waku_filter,
-  ../test_helpers, ./utils
+  ../../waku/v2/protocol/waku_message,
+  ../../waku/v2/protocol/waku_filter,
+  ../test_helpers, 
+  ./utils
 
-procSuite "Waku Filter":
 
-  asyncTest "handle filter":
-    const defaultTopic = "/waku/2/default-waku/proto"
+const 
+  DEFAULT_PUBSUB_TOPIC = "/waku/2/default-waku/proto"
+  DEFAULT_CONTENT_TOPIC = ContentTopic("/waku/2/default-content/proto")

```

Comment:
CamelCase, perhaps? Although we can re-discuss our style choice here if you want, as long as we come up with something consistent and Nim-like for `const` :)

Reply:
Oh, man... Sometimes I go in auto-pilot mode and I forget about those things, sorry 😅 Changing it!